lib/upgrade_format_task.rb

```diff
@@ -52,12 +58,10 @@ def upgrade(store, name, options = {})
 
     if options[:all]
       if migrations_done > 0
-        Machinery::Ui.puts "Upgraded #{migrations_done} system descriptions successfully."
-      else
+        Machinery::Ui.puts "Upgraded #{migrations_done} #{"system description".pluralize(migrations_done)}."
+      elsif migrations_done = 0
```

Comment:
This is using the assignment operator `=`, not the comparison operator `==`, so it actually doesn't check the condition but sets `migrations_done` to 0. This is not what is supposed to happen, although it doesn't have any effect here.

Actually I would leave out the condition and just leave the plain `else`. This way the if statement covers all cases.